### DISCUSSION TITLE
Replace the argument "model" with "weights" in segmentation training usage example

### DISCUSSION THREAD

**Discussion by memelvin** [ultralytics/yolov5#10287] [General]
Hi,
one small typo correction in the segmentation training usage example in the main page of yolov5 repo. Please replace "model" argument with "weights". Otherwise getting an error :> _train.py: error: unrecognized arguments: --model yolov5s-seg.pt_

![image](https://user-images.githubusercontent.com/44300707/203928774-e7bcc392-d489-424b-8ab1-e60e1ce2ea86.png)

**Comment by glenn-jocher** [comment]
@memelvin good news 😃! Your original issue may now be fixed ✅ in PR #10298. 

To receive this update:
- **[Git](https://github.com/ultralytics/yolov5)** – `git pull` from within your `yolov5/` directory or `git clone https://github.com/ultralytics/yolov5` again
- **[PyTorch Hub](https://pytorch.org/hub/ultralytics_yolov5/)** – Force-reload `model = torch.hub.load('ultralytics/yolov5', 'yolov5s', force_reload=True)`
- **[Notebooks](https://github.com/ultralytics/yolov5/blob/master/tutorial.ipynb)** – View updated notebooks  <a href="https://bit.ly/yolov5-paperspace-notebook"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"></a> <a href="https://colab.research.google.com/github/ultralytics/yolov5/blob/master/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/ultralytics/yolov5"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **[Docker](https://hub.docker.com/r/ultralytics/yolov5)** – `sudo docker pull ultralytics/yolov5:latest` to update your image <a href="https://hub.docker.com/r/ultralytics/yolov5"><img src="https://img.shields.io/docker/pulls/ultralytics/yolov5?logo=docker" alt="Docker Pulls"></a>

Thank you for spotting this issue and informing us of the problem. Please let us know if this update resolves the issue for you, and feel free to inform us of any other **issues** you discover or **feature requests** that come to mind. Happy trainings with YOLOv5 🚀!
